### DISCUSSION TITLE
wiki request

### DISCUSSION THREAD

**Discussion by lvjiahui** [BoomingTech/Piccolo#236] [Ideas]
wiki对阅读代码非常有帮助，求更~

**Comment by hyv1001** [comment]
感谢！
当前团队一方面在重构一些框架上内容，包括基础的Level，Object，Component系统，以及Rendering，Animation，Physics等系统，主要是让代码和架构更容易理解。这周已经发布了部分内容： #177 #179 #181 ，下周也会继续更新 Rendering，Physics相关的系统，后续也会把相关文档逐步补充完整。
